stubs/tensorflow/tensorflow/config/experimental.pyi

```diff
@@ -0,0 +1,15 @@
+from _typeshed import Incomplete
+from typing import TypedDict
+
+from tensorflow.config import PhysicalDevice
+
+class _MemoryInfo(TypedDict):
+    current: int
+    peak: int
+
+def get_memory_info(device: str) -> _MemoryInfo: ...
+def reset_memory_stats(device: str) -> None: ...
+def get_memory_usage(device: PhysicalDevice) -> int: ...
```

Comment:
Looks like this is deprecated:

```
@deprecation.deprecated(
    None,
    "Use tf.config.experimental.get_memory_info(device)['current'] instead.")
```

Consider marking it with `@typing_extensions.deprecated`.

Reply:
Done in 9898dc763